draco/asp/soft.lp

```diff
@@ -30,3 +30,18 @@ preference(same_field,F) :-
     entity(mark,_,M),
     { entity(encoding,M,E): attribute((encoding,field),E,F) } = 2.
 
+% @soft(type_linear) Prefer linear=log>ordinal>categorical
```

Comment:
I think in draco1 these are about data types and yours are scale types

Reply:
Yes. We don't need endowing types anymore.